mk-files/release.mk

```diff
@@ -149,7 +149,8 @@ endef
 .PHONY: download-licenses
 download-licenses:
 	go install github.com/google/go-licenses@v1.5.0 && \
-	go-licenses save ./... --save_path legal/licenses --force || true
+	go-licenses save ./... --save_path legal/licenses/ --force || true && \
+	chmod -R +w legal/licenses/
```

Comment:
I don't understand this line's purpose but otherwise lgtm

Reply:
The `go-licenses` package creates a read-only directory, so when you try to delete it with `make clean` it spams a bunch of errors to the screen.

Reply:
Oh. That's annoying.